p/themes/Origine/origine.css

```diff
@@ -140,27 +145,52 @@ form th {
 	border-radius: 0 3px 3px 0;
 }
 
-.stick .btn + .btn,
-.stick .btn + input,
-.stick .btn + .dropdown > .btn,
-.stick input + .btn,
-.stick input + input,
-.stick input + .dropdown > .btn,
-.stick .dropdown + .btn,
-.stick .dropdown + input,
-.stick .dropdown + .dropdown > .btn {
+html.rtl .stick .btn:last-child,
+html.rtl .stick input:last-child {
+	border-radius: 3px 0 0 3px;
+}
+
+html:not(.rtl) .stick .btn + .btn,
```

Comment:
Can't we avoid the `:not(.rtl)` thanks to the `.rtl` below, overriding the properties?

Reply:
Not quite elegantly. Options include `border-left-width: 0;` instead of none, so you can easily restore it to some `px` value (probably the best alternative) and possibly also some invalid value.

Reply:
Ok

Reply:
The alternative looks like this:

```diff
diff --git a/p/themes/Origine/origine.css b/p/themes/Origine/origine.css
index 86210940..5f90fb1b 100644
--- a/p/themes/Origine/origine.css
+++ b/p/themes/Origine/origine.css
@@ -150,16 +150,16 @@ html.rtl .stick input:last-child {
        border-radius: 3px 0 0 3px;
 }
 
-html:not(.rtl) .stick .btn + .btn,
-html:not(.rtl) .stick .btn + input,
-html:not(.rtl) .stick .btn + .dropdown > .btn,
-html:not(.rtl) .stick input + .btn,
-html:not(.rtl) .stick input + input,
-html:not(.rtl) .stick input + .dropdown > .btn,
-html:not(.rtl) .stick .dropdown + .btn,
-html:not(.rtl) .stick .dropdown + input,
-html:not(.rtl) .stick .dropdown + .dropdown > .btn {
-       border-left: none;
+.stick .btn + .btn,
+.stick .btn + input,
+.stick .btn + .dropdown > .btn,
+.stick input + .btn,
+.stick input + input,
+.stick input + .dropdown > .btn,
+.stick .dropdown + .btn,
+.stick .dropdown + input,
+.stick .dropdown + .dropdown > .btn {
+       border-left-width: 0;
 }
 
 html.rtl .stick .btn + .btn,
@@ -171,7 +171,8 @@ html.rtl .stick input + .dropdown > .btn,
 html.rtl .stick .dropdown + .btn,
 html.rtl .stick .dropdown + input,
 html.rtl .stick .dropdown + .dropdown > .btn {
-       border-right: none;
+       border-left-width: 1px;
+       border-right-width: 0;
 }
 
 .stick input + .btn {
```

Reply:
It looks a bit lighter, maybe? No strong preference

Reply:
I went with the current version because I figured it's slightly easier to parse (if not ltr then border-left:none, elseif ltr then border-right: none) but I'm fine with any approach. A similar option is to add `html.ltr` as well.